packages/account-v2/package.json

```diff
@@ -0,0 +1,60 @@
+{
+    "name": "@deriv/account-v2",
+    "version": "2.0.0",
+    "main": "dist/account-v2/js/index.js",
+    "engines": {
+        "node": "18.x"
+    },
+    "scripts": {
+        "build": "rimraf dist && NODE_OPTIONS='-r ts-node/register' webpack --progress --config \"./webpack.config.js\" && npm run translate",
+        "serve": "rimraf dist && concurrently \"cross-env BUILD_MODE='serve' NODE_OPTIONS='-r ts-node/register' webpack --progress --watch --config ./webpack.config.js\" \"tsc -w --noEmit --preserveWatchOutput\"",
+        "start": "rimraf dist && npm run test && npm run serve",
+        "translate": "sh ./scripts/update-translations.sh"
+    },
+    "dependencies": {
+        "@deriv/api": "^1.0.0",
+        "@deriv/quill-design": "^1.2.12",
+        "classnames": "^2.3.2",
+        "i18n-iso-countries": "^7.7.0",
+        "i18next": "^23.7.11",
+        "moment": "^2.29.4",
+        "qrcode.react": "^3.1.0",
+        "react": "^17.0.2",
+        "react-dom": "^17.0.2",
+        "react-dropzone": "11.0.1",
+        "react-i18next": "^11.11.0",
+        "react-router-dom": "^5.2.0",
+        "react-transition-group": "4.4.2",
+        "usehooks-ts": "^2.7.0"
+    },
+    "devDependencies": {
+        "@testing-library/react": "^12.0.0",
+        "@testing-library/user-event": "^13.5.0",
+        "@types/css-modules": "^1.0.2",
+        "@types/react-dom": "^18.0.0",
+        "@typescript-eslint/eslint-plugin": "5.45.0",
+        "@typescript-eslint/parser": "5.45.0",
+        "autoprefixer": "^10.4.16",
+        "eslint": "^8.56.0",
+        "eslint-config-airbnb": "^19.0.4",
+        "eslint-config-prettier": "^9.0.0",
+        "eslint-plugin-import": "^2.29.1",
+        "eslint-plugin-jsx-a11y": "^6.8.0",
+        "eslint-plugin-local-rules": "2.0.0",
+        "eslint-plugin-prettier": "^5.0.0",
+        "eslint-plugin-react": "^7.33.2",
+        "eslint-plugin-react-hooks": "^4.6.0",
+        "eslint-plugin-simple-import-sort": "^10.0.0",
+        "eslint-plugin-sort-destructure-keys": "^1.5.0",
+        "eslint-plugin-tailwindcss": "^3.13.0",
+        "eslint-plugin-typescript-sort-keys": "^2.3.0",
+        "postcss": "^8.4.24",
+        "stylelint": "^16.0.2",
+        "stylelint-config-standard": "^35.0.0",
+        "tailwindcss": "^3.3.6",
+        "typescript": "^4.6.3",
```

Comment:
why not the lates TS?

Reply:
Cant have a mismatch version with hoisted packages